frontend/src/components/dialogs/contributor-dialog.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <base-dialog :show="opened" title="Add contributor">
+    <div>
+      <slot name="userName"></slot>
+    </div>
+    <p v-if="emptyUserName">Incorrect user name. Please, enter the valid one.</p>
```

Comment:
If I understand correctly, you are showing error if input is incorrect (i refer frontend/src/pages/documents/documents.page.vue). It can be done in MUUUCH simplier way:
create ```InputValidation``` component which accepts label and ```validate: (input: string) => Error | null```. Then manage error displaying in ```InputValidation```